edgedb/datatypes/object.c

```diff
@@ -216,28 +216,83 @@ static PyObject *
 object_getitem(EdgeObject *o, PyObject *name)
 {
     Py_ssize_t pos;
+    int prefixed = 0;
+    if (PyUnicode_Check(name)) {
+        PyObject *at_sign = PyUnicode_FromString("@");
```

Comment:
Can you cache this object globally? (Only if it's easy and we can use some safe CPython API for global module state)